test/JDBC/input/functions/string_agg-vu-verify.sql

```diff
@@ -0,0 +1,137 @@
+-- expression as column
+SELECT STRING_AGG(a,'-') FROM string_agg_t
+GO
+
+SELECT STRING_AGG(a,'-') FROM string_agg_t GROUP BY g
+GO
+
+SELECT STRING_AGG(a,'-') WITHIN GROUP (ORDER BY sbid) FROM string_agg_t GROUP BY g
+GO
+
+SELECT STRING_AGG(a,'-') WITHIN GROUP (ORDER BY sbid ASC) FROM string_agg_t GROUP BY g
+GO
+
+SELECT STRING_AGG(a,'-') WITHIN GROUP (ORDER BY sbid DESC) FROM string_agg_t GROUP BY g
+GO
+
+SELECT STRING_AGG(a,'-') WITHIN GROUP (ORDER BY id, sbid ASC) FROM string_agg_t GROUP BY g
+GO
+
+SELECT STRING_AGG(a,'-') WITHIN GROUP (ORDER BY id, sbid DESC) FROM string_agg_t GROUP BY g
+GO
+
+-- expression as expression of multiple columns
+SELECT STRING_AGG(a+b,'-') FROM string_agg_t
+GO
+
+SELECT STRING_AGG(a+b,'-') FROM string_agg_t GROUP BY g
+GO
+
+SELECT STRING_AGG(a+b,'-') WITHIN GROUP (ORDER BY sbid) FROM string_agg_t GROUP BY g
+GO
+
+SELECT STRING_AGG(a+b,'-') WITHIN GROUP (ORDER BY sbid ASC) FROM string_agg_t GROUP BY g
+GO
+
+SELECT STRING_AGG(a+b,'-') WITHIN GROUP (ORDER BY sbid DESC) FROM string_agg_t GROUP BY g
+GO
+
+SELECT STRING_AGG(a+b,'-') WITHIN GROUP (ORDER BY id, sbid ASC) FROM string_agg_t GROUP BY g
+GO
+
+SELECT STRING_AGG(a+b,'-') WITHIN GROUP (ORDER BY id, sbid DESC) FROM string_agg_t GROUP BY g
+GO
+
+-- expression as function
+SELECT STRING_AGG(concat(a,b),'-') FROM string_agg_t
+GO
+
+SELECT STRING_AGG(concat(a,b),'-') FROM string_agg_t GROUP BY g
+GO
+
+SELECT STRING_AGG(concat(a,b),'-') WITHIN GROUP (ORDER BY sbid) FROM string_agg_t GROUP BY g
+GO
+
+SELECT STRING_AGG(concat(a,b),'-') WITHIN GROUP (ORDER BY sbid ASC) FROM string_agg_t GROUP BY g
+GO
+
+SELECT STRING_AGG(concat(a,b),'-') WITHIN GROUP (ORDER BY sbid DESC) FROM string_agg_t GROUP BY g
+GO
+
+SELECT STRING_AGG(concat(a,b),'-') WITHIN GROUP (ORDER BY id, sbid ASC) FROM string_agg_t GROUP BY g
+GO
+
+SELECT STRING_AGG(concat(a,b),'-') WITHIN GROUP (ORDER BY id, sbid DESC) FROM string_agg_t GROUP BY g
+GO
+
+-- Delimeter as a function
+SELECT STRING_AGG(a, char(10)) FROM string_agg_t
+GO
+
+SELECT STRING_AGG(a, char(10)) FROM string_agg_t GROUP BY g
+GO
+
+SELECT STRING_AGG(a, char(10)) WITHIN GROUP (ORDER BY sbid) FROM string_agg_t GROUP BY g
+GO
+
+SELECT STRING_AGG(a, char(10)) WITHIN GROUP (ORDER BY sbid ASC) FROM string_agg_t GROUP BY g
+GO
+
+SELECT STRING_AGG(a, char(10)) WITHIN GROUP (ORDER BY sbid DESC) FROM string_agg_t GROUP BY g
+GO
+
+SELECT STRING_AGG(a, char(10)) WITHIN GROUP (ORDER BY id, sbid ASC) FROM string_agg_t GROUP BY g
+GO
+
+SELECT STRING_AGG(a, char(10)) WITHIN GROUP (ORDER BY id, sbid DESC) FROM string_agg_t GROUP BY g
+GO
+
+-- Batch statements
+SELECT STRING_AGG(a, char(10)) WITHIN GROUP (ORDER BY sbid) FROM string_agg_t GROUP BY g
+SELECT STRING_AGG(a, char(10)) WITHIN GROUP (ORDER BY sbid ASC) FROM string_agg_t GROUP BY g
```

Comment:
Also can you add a few more test cases for STRING_AGG() with Multibyte Characters.

Reply:
Done